Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
>  In one of my projects I get garbled sming debug output, possible caused by a wrong linker setting.

Make sure to initialize the flash memory when changing memory layouts or/and SDKs: 
https://github.com/SmingHub/Sming/wiki/Application%20prints%20random%20symbols%20and%20restarts